src/test/java/com/wikia/webdriver/elements/fandom/components/Notification.java

```diff
@@ -0,0 +1,32 @@
+package com.wikia.webdriver.elements.fandom.components;
```

Comment:
Change directory of this class. As Notifications functionality is part of global navigation component I'd suggest `/elements/oasis/notifications`. (Did you noticed that there is already `BannerNotifications` class? If this is the same functionality then we should merge your changes and already existing class.